clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -45,11 +45,11 @@ public class CommonClientConfigs {
      */
 
     public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
-    public static final String BOOTSTRAP_SERVERS_DOC = "A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. The client will make use of all servers irrespective of which servers are specified here for bootstrapping&mdash;this list only impacts the initial hosts used to discover the full set of servers. This list should be in the form "
-                                                       + "<code>host1:port1,host2:port2,...</code>. Since these servers are just used for the initial connection to "
-                                                       + "discover the full cluster membership (which may change dynamically), this list need not contain the full set of "
-                                                       + "servers (you may want more than one, though, in case a server is down).";
```

Comment:
I wonder if we could just change `you may want` to `we recommend using`

Reply:
Is it really important to mention clients randomly pick an entry in the list when first bootstrapping?

Reply:
I've updated and think the description reads well now, but let me know if I've interpreted your comments incorrectly.